sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 2.0.0-beta.1 (2024-05-07)
 
 ### Features Added
+- `documenttranslate` is the method added to submit a single/synchronous document translation request to the Document Translation service.
```

Comment:
the name of the method should match the name that's in this client library.

Reply:
Updated